TMEDIA 725: Convert Footer block (#1496)
WPMedia/arc-themes-blocks: blocks/footer-block/README.md

## Internationalization fields

- rss-link
- facebook-link

> - twitter-link

Reviewer: We usually pair these in a markdown table with"default (English)" examples.

> - twitter-link

Reviewer: We usually pair these in a markdown table with"default (English)" examples.

## Events

> 
> n/a
> 
> ### Event Listening
> 
> This block does not emit any events.

Reviewer: ```suggestion
This block does not emit or listen for events.
```

> 
> ## Additional Considerations

Reviewer: ```suggestion
```

Reviewer: We can probably take this out entirely.
